subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings.c

```diff
@@ -45,6 +45,9 @@ struct backend_data_t {
 	/* General */
 	unsigned int role;
 	atomic_t state;
+
+	/* TX buffer size */
+	int tx_buffer_size;
```

Comment:
Does it necessary to have signed type for buffer size instead of for example `size_t` type ?